### PR TITLE
Problems in remote deployment #2337

### PR DESCRIPTION
**Pull request by micgibso**
Signed-off-by: micgibso <MICGIBSO@uk.ibm.com>

PR for issue https://github.com/eclipse/codewind/issues/2337

### CODE REVIEW

**Comment by micgibso** [comment]
@markcor11 Thanks for the discussion, all comments now resolved and applied to this PR for your review.

File: docs/_documentations/remotedeploy-existingkeycloak.md [hunk 1]
```diff
@@ -116,6 +114,7 @@ This command performs the following actions:
 - Creates an initial Codewind user called `developer` with password `passw0rd`.
 - Uses the ingress appropriate to the deployment environment.
 - Adds the Keycloak admin credentials of `admin` with password `passw0rd`.
```
Comment (markcor11) [inline docs/_documentations/remotedeploy-existingkeycloak.md]: The kadminuser and kadminpass dont so much "add" an admin user to keycloak (that is done separately when Keycloak is first installed).  Here, kadminuser and kadminpass are required by CWCTL to login to the already existing Keycloak in order to configure it with the realm/client/developer user items.

Comment (micgibso) [reply docs/_documentations/remotedeploy-existingkeycloak.md]: Updated sentence to make it clear we're not setting the credentials here.

**Comment by micgibso** [comment]
@markcor11 Change made, please review, thx!

**Pull request merged**